git_machete/client.py

```diff
@@ -2362,3 +2386,23 @@ def __sync_before_creating_pr(self, *, opt_onto: Optional[LocalBranchShortName],
             if ans in ('y', 'yes'):
                 return
             raise MacheteException('Pull request creation interrupted.')
+
+    def delete_untracked(self,
+                         opt_yes: bool,
+                         opt_merge: bool,
+                         opt_no_interactive_rebase: bool,
+                         opt_no_edit_merge: bool) -> None:
+        untracked_branches: List[LocalBranchShortName] = []
+        for managed_branch in self.managed_branches:
+            status, _ = self.__git.get_strict_remote_sync_status(managed_branch)
+            if status == SyncToRemoteStatuses.UNTRACKED:
+                untracked_branches.append(managed_branch)
+
+        if untracked_branches:
+            self._slide_out_branches_from_git_machete(branches_to_slide_out=untracked_branches,
```

Comment:
AFAICS `_slide_out_branches_from_git_machete` has some extra requirements getting checked, like:

```
        # Verify that all "interior" slide-out branches have a single downstream
        # pointing to the next slide-out
```

that make sense for `slide-out` command, but might not be the best choice here 🤔

Reply:
Why is it not the best choice here?

Reply:
Pls add a test case where `clean` is executed` against the following tree:

```
master
|
o-foo (untracked)
|  |
|  o-foo2
|
o-bar (untracked)
  |
  o-bar2
```

I'm afraid this will not work as expected!